src/components/input-message/input-message.tsx

```diff
@@ -26,7 +26,11 @@ export class InputMessage {
   //
   //--------------------------------------------------------------------------
 
-  /** When `true`, the component is active. */
+  /**
+   * When `true`, the component is active.
+   *
+   * @deprecated use global `hidden` attribute instead.
```

Comment:
nitpick: Should we add the link to the `mdn` hidden attribute here? Not necessary but might be nice.

Reply:
I think that'd be great. I'll add it now.